src/app/onto-search/onto-search.component.ts

```diff
@@ -0,0 +1,129 @@
+import {Component, EventEmitter, Input, OnInit, Output, TemplateRef} from '@angular/core';
+import {Observable, of} from 'rxjs';
+import {SearchModel} from 'src/app/onto-search/models/search-model';
+
+@Component({
+  selector: 'onto-search',
+  templateUrl: './onto-search.component.html',
+  styleUrls: ['./onto-search.component.scss'],
+})
+export class OntoSearchComponent implements OnInit {
```

Comment:
I think the component should have an input for search terms. If we resolve search terms from the url parameters, there should be a way to populate the chip-list.